crates/ursa/src/ursa/rpc_commands.rs

```diff
@@ -34,7 +32,11 @@ impl RpcCommands {
                 let params = NetworkPutFileParams {
                     path: path.to_string(),
                 };
-                match put_file(params, *port).await {
+                let mut client = Client::default();
```

Comment:
Shouldn’t we initiate client per rpc service and not per command. Also noticed we are still passing rpc port in reach command

Reply:
Since the feedback on #158 was on parameters of rpc client functions #158, command refactoring was not addressed here :(. I can open an issue for removing the ports though.

Reply:
https://github.com/fleek-network/ursa/issues/179

Reply:
This issue was introduced in #156 where the initial feedback was given